config-x86-gpu.ini

```diff
@@ -73,28 +71,47 @@ ClassID = 1
 MinScore = 0.25
 
 [Classifier]
+Device = x86-gpu
 Name = OFMClassifier
 ModelPath =
 ImageSize = 45,45,3
 
-[PostProcessor]
-; Number of times tracker was lost while tracking
+[Tracker]
+Name = IOUTracker
 MaxLost = 5
 TrackerIOUThreshold = 0.5
+
+[SourcePostProcessor_0]
+Name = objects_filtering
 NMSThreshold = 0.98
-; Default distance measurement method (used when no DistMethod is specified for the source):
```

Comment:
I think the comments are useful and we should try to keep them. Same for the comments that mention which models are available